docs-ref-conceptual/azure-cli-reference-for-IoT.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Azure CLI references for Azure IoT
+description: Azure CLI reference landing page for Azure IoT
+author: dbradish-microsoft
+manager: barbkess
+ms.devlang: azurecli
+ms.topic: reference
+ms.date: 06/05/2020
+ms.author: dbradish
+ms.service: azure-cli
+ms.devlang: azurecli
+---
+
+# Azure CLI for Azure IoT
+
+The Azure Command Line Interface ([Azure CLI](/cli/azure/what-is-azure-cli)) is a set of commands used to create and manage Azure resources.  It is available across many Azure services including Azure IoT.  There are over 100 references for Azure IoT giving you the ability to work effectively with IoT services from a command line.
+
+## References for IoT
+
+The Azure IoT CLI experience is composed of two parts: Azure CLI (commonly referred to as CLI **core**) and the Azure IoT CLI **extension**.
+
+IoT functionality in Azure CLI **core** is focused on infrastructure management and configuration. IoT Hub CRUD operations, or configuring IoT Hub message routes are typical use cases for core commands.
+
+The IoT **extension** introduces rich features and functionality.  Extensions manage, manipulate and interact with the data, entities and objects on the infrastructure itself. Managing fleets of devices, monitoring device-to-cloud events and invoking cloud to device methods are all enabled via the IoT extension. Azure CLI extensions for IoT unlock the use of experimental or pre-release technology contributing to its versatility in a variety of scenarios and use cases.
```

Comment:
The second sentence where it starts with ". Extensions manage ..." - this description is specific to the IoT extension and there is only 1 certified IoT extension by Azure IoT. Also these are examples of functional categories of what the IoT extension can do, not an exhaustive list, which is why I think we need a "For example" disclaimer on there.

Refactor suggestion:
`
The IoT **extension** introduces rich features and functionality to manage, manipulate and interact with the data, entities and objects on the infrastructure itself. For example managing fleets of devices, monitoring device-to-cloud events and invoking cloud to device methods are all enabled via the IoT extension. The Azure IoT extension for Azure CLI unlocks the use of experimental or pre-release technology contributing to its versatility in a variety of scenarios and use cases.
`